CHANGELOG.asciidoc

```diff
@@ -0,0 +1,667 @@
+ifdef::env-github[]
+NOTE: Release notes are best read in our documentation at
+https://www.elastic.co/guide/en/apm/agent/nodejs/current/release-notes.html[elastic.co]
+endif::[]
+
+////
+[[release-notes-x.x.x]]
+=== x.x.x - YYYY/MM/DD
+
+[float]
+==== Features
+* Cool new feature: {pull}2526[#2526]
+
+[float]
+==== Bug fixes
+
+[float]
+==== Breaking changes
+////
+
+[[release-notes-3.0.0]]
+=== 3.0.0 - 2019/9/30
+
+[float]
+==== Bug fixes
+* feat: allow manual instrumentation with `instrument: false` {pull}1114[#1114]
+* feat: allow setting span/transaction `type`, `subtype`, and `action` separately (the behavior of the old `type` has changed) {pull}1292[#1292]
+* feat: use `external` as span type instead of `ext` {pull}1291[#1291]
+* refactor(graphql): use custom transaction type `graphql` for graphql requests instead of `request` {pull}1245[#1245]
+* feat(http): add `instrumentIncomingHTTPRequests` config (`disableInstrumentations` now behaves differently) {pull}1298[#1298]
+* chore: remove deprecated APIs {pull}1413[#1413]
+* chore: drop support for older Node.js versions {pull}1383[#1383]
+
+[[release-notes-2.17.1]]
+=== 2.17.1 - 2019/9/26
+
+[float]
+==== Bug fixes
+* fix: support all falsy return values from error filters {pull}1389[#1389]
+* fix: capture all non-string http bodies {pull}1376[#1376]
+
+[[release-notes-2.17.0]]
+=== 2.17.0 - 2019/9/19
+
+[float]
+==== Features
+* feat: add support for @koa/router {pull}1346[#1346]
+* feat: add methods for logging trace information {pull}1335[#1335]
+
+[float]
+==== Bug fixes
+* fix: improve debug output when detecting incoming http request {pull}1357[#1357]
+* fix(http): response context propagation on Node.js 12.0 - 12.2 {pull}1339[#1339]
+
+[[release-notes-2.16.2]]
+=== 2.16.2 - 2019/9/3
+* fix(lambda): handle traceparent case-insensitively {pull}1319[#1319]
+
+[[release-notes-2.16.1]]
+=== 2.16.1 - 2019/8/28
+* fix: avoid throwing when agent is in active: false mode {pull}1278[#1278]
```

Comment:
We should probably leave the commit grouping headers in, even when there's only one type, just to be clear.